Web/Edubase.Web.UI/Areas/Groups/Controllers/GroupController.cs

```diff
@@ -16,8 +16,10 @@
 namespace Edubase.Web.UI.Areas.Groups.Controllers
 {
     using Common;
+    using Edubase.Web.UI.Areas.Groups.ViewRulesHandlers;
     using Exceptions;
     using Filters;
+    using global::Glimpse.Mvc.Model;
```

Comment:
IDE reports this as a newly added unused import. If this is correct, let's remove it?

Reply:
nor sure where this is from removed it